active/website.md

```diff
@@ -0,0 +1,61 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Members triage the project’s issues and pull requests.
+- Member maintain and monitor the website including updating versions.
+- Mentor new contributors to the website.
+- Run or support djangoproject.com sessions in DjangoCon sprints.
+- Chair, Co-Chair and Board Liaison can sign off on new features.
+
+## Initial membership
+
+- Chair: Sarah Abderemane
+- Co-Chair: Saptak Sengupta
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Sarah Abderemane
+- Other members:
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 
+
+### How will decisions on adding/removing members be handled?
```

Comment:
Missing removing members.

Reply:
I think after the one year term, anyone who doesn't opt-in to continue being involved in the WG can be removed. (We can also do an opt-out instead of opt-in mechanism. I don't have a super strong feeling about either). Also, I feel, maybe violation of code of conduct can result in removal of a member?

Reply:
> Also, I feel, maybe violation of code of conduct can result in removal of a member?

I think that's implied generally, the CoC-WG (and the Board) has the power to impose various sanctions, including removing from WGs if necessary. So you don't need to explicitly say it here.

Reply:
Resolving to keeping the one year opt-in option.